init.lua

```diff
@@ -134,10 +134,6 @@ vim.opt.signcolumn = 'yes'
 -- Decrease update time
 vim.opt.updatetime = 250
 
--- Decrease mapped sequence wait time
--- Displays which-key popup sooner
-vim.opt.timeoutlen = 300
```

Comment:
Keep this in. Removing this increases the timeoutlen to the default 1000 ms for all other map sequences.

Reply:
Fixed.